rever/activities/travis_ci.xsh

```diff
@@ -0,0 +1,99 @@
+"""Activity for keeping a changelog from news entries."""
+import os
+import re
+import sys
+
+from xonsh.tools import print_color
+
+from rever import vcsutils
+from rever.activity import Activity
+from rever.tools import eval_version, replace_in_file
+
+TRAVIS_CI = """sudo: False
+
+language: python
+
+matrix:
+  include:
+    - python: 3.6
+
+install:
+  # Install conda
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set always_yes yes --set changeps1 no
+  - conda config --add channels conda-forge
+  - conda update conda
+
+  # Install dependencies
+  - conda install {conda_deps} -c (conda_channels)
+  - pip install {pip_deps}
+  - python setup.py install
+
+# script:
+  # - set -e
+  # - # Command to build your docs
+  # - pip install doctr
```

Comment:
We may want to remove doctr from this for now, although it would be nice to have a doctr setup activity which we could depend on.